api/asyncapi/websocket.yml

```diff
@@ -0,0 +1,88 @@
+asyncapi: "2.6.0"
```

Comment:
Seems that this doc is not valid when I test in Async API [editor](https://studio.asyncapi.com/#schema-jsonMsg). Please check.